.travis.yml

```diff
@@ -42,11 +42,10 @@ addons:
 
 install:
   - pip install -U pip
-  - pip install -U tox-travis
+  - pip install -U tox-travis tox-conda
```

Comment:
After the fix, do we still need `tox-conda`?

Reply:
Yes, it's being used to install the `pybind11` conda package in the same env (also `ninja`)